packages/app-settings/src/Developer.tsx

```diff
@@ -14,7 +14,16 @@ import { isJsonObject, stringToU8a, u8aToString } from '@polkadot/util';
 import { useTranslation } from './translate';
 
 const EMPTY_CODE = '{\n\n}';
-const EMPTY_TYPES = {};
+// Adjusted type definitions which allow
+// UI to render NFT components nicely
```

Comment:
may be we don't need this now.. but we still need the fix in the file above (TransferWithType)